about/press.html

```diff
@@ -101,14 +101,67 @@
       <h3>Contact</h3>
       <p>For press inquiries, send us an email to <a href="mailto:summer-of-code@railsgirls.com">summer-of-code@railsgirls.com</a>.</p>
 
+      <br>
+      <h2>Press Releases and Media</h2>
+
+      <p>We gathered on the following sections a group of resources, such as press releases, logos, images, videos, quotes and some of the articles written about the program. Please feel free to use them, and in order to make the download easier, you can find the whole press kit <a href="https://www.dropbox.com/sh/ng9p8d5hba425ht/AAB18_F_NZzX_UlQL8LB1caMa?dl=0">here</a>.</p>
```

Comment:
just fixing some wording, can you change to:
"In the following sections, we gathered a selection of resources.."
and 
"Please feel free to use them. In order to..."